scripts/update_swagger.py

```diff
@@ -22,16 +22,19 @@
 http = http_client(log)
 
 repository_url = 'https://raw.githubusercontent.com/swagger-api/swagger-ui'
-tag = 'v5.28.1'
+tag = 'v5.29.3'
```

Comment:
```suggestion
tag = 'v5.29.4'
```

https://github.com/DataBiosphere/azul/issues/7468#issuecomment-3391429053